packages/taquito-rpc/src/types.ts

```diff
@@ -596,24 +598,26 @@ export interface ConstantsResponse {
   tokens_per_roll: BigNumber;
   michelson_maximum_type_size: number;
   seed_nonce_revelation_tip: string;
-  origination_burn: string;
+  origination_size: number;
   block_security_deposit: BigNumber;
   endorsement_security_deposit: BigNumber;
-  block_reward: BigNumber;
```

Comment:
Removing this field will be problematic for protos < 7.

Reply:
This will be resolved by adding union type (in the following commit)